airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/client/api.py

```diff
@@ -154,9 +154,10 @@ def read(self, getter: Callable, params: Mapping[str, Any] = None) -> Iterator:
         """Apply state filter to set of records, update cursor(state) if necessary in the end"""
         params = params or {}
         latest_cursor = None
+        buffer_days = getattr(self, 'buffer_days') + 1 if hasattr(self, 'buffer_days') else 0
```

Comment:
```suggestion
        buffer_days = getattr(self, 'buffer_days', 0) + 1
```

Reply:
@keu, I think the logic will be a little wrong with your changes

Reply:
@yevhenii-ldv it was a test :) ok, I have updated

Reply:
@vitaliizazmic I have changed my mind, lets add loop_back=0 to IncrementalStream, and just use normal self.loop_back instead

Reply:
the problem with the current approach is that base class assume implementation of its child classes, which actually should wise versa

Reply:
also my bad, I mean `lookback` or `buffer_days`